android/capacitor/src/test/java/com/getcapacitor/JSObjectTest.java

```diff
@@ -0,0 +1,208 @@
+package com.getcapacitor;
+
+import org.json.JSONException;
+import org.junit.Assert;
```

Comment:
This can be changed to `import static org.junit.Assert.*;` then all the `Assert.assertFoo(...)` can become `assertFoo(...)`

Reply:
Thanks, done!